src/Entities/Assertions/AssertionFactory.php

```diff
@@ -75,6 +76,11 @@ class AssertionFactory
         'void'
     );
 
+    /**
```

Comment:
Please extend the docblock to be phpdocumentor usable. Should at least contain description, type and proeprty name (see other properties for example)